src/bin/bpf-linker.rs

```diff
@@ -110,8 +152,8 @@ struct CommandLine {
     log_file: Option<PathBuf>,
 
     /// Set the log level. Can be one of `off`, `info`, `warn`, `debug`, `trace`.
```

Comment:
Is this doc comment still accurate? I don't see `off` in the enum + `error` is missing from the doc comment.

Reply:
Thanks for pointing out, it's inaccurate. I removed `off` from the comment.